radio/src/serial.cpp

```diff
@@ -304,9 +332,8 @@ void serialInit(uint8_t port_nr, int mode)
         state->usart_ctx = port->uart->init(&params);
 
       // Set power on/off
-      if (port->set_pwr) {
-        port->set_pwr(power_required);
-      }
+      if (port_nr < SP_VCP)
```

Comment:
If you wrap this with `#if defined(SWSERIALPOWER)` this should implicity optimise `serialSetPowerState`, `getSerialPower`, `setSerialPower` etc out on radios that don't have this... only save about 64 bytes firmware size, but hey, it's a 'free' saving :)

Reply:
Thx for the tip, fixed in latest commit.